src/Exercise9.js

```diff
@@ -0,0 +1,65 @@
+import React, { useCallback, useEffect, useState } from "react";
+import ReactFlipCard from "reactjs-flip-card";
+
+const BackFace = () => {
+  return <div className="text-xl font-extrabold">More information</div>;
+};
+
```

Comment:
You can improve this function to not have the return function to be like this: 
```suggestion
const BackFace = () => (
  <div className="text-xl font-extrabold">More information</div>
);
```